server/src-lib/Hasura/RQL/DDL/Metadata.hs

```diff
@@ -147,8 +164,9 @@ runClearMetadata _ = do
 
 data ReplaceMetadata
   = ReplaceMetadata
-  { aqTables           :: ![TableMeta]
-  , aqFunctions        :: !(Maybe [QualifiedFunction])
+  { aqVersion          :: !(Maybe MetadataVersion)
+  , aqTables           :: ![TableMeta]
+  , aqFunctions        :: !(Maybe [Value])
```

Comment:
This means we’ll insert absolutely any arbitrary JSON value into the database, right? That seems like it’s probably too permissive, since we don’t allow loading arbitrary JSON values out of the database, right? It seems like a user could end up with their database in an invalid state that way.

Reply:
It is really odd to have `Value` here. Just now, I realised `parse, don't validate!` :smile: . As in I use `Value` here to validate it later. I'll make the necessary change.